### DISCUSSION TITLE
Disable minification?

### DISCUSSION THREAD

**Discussion by akfaew** [a-h/templ#970] [Ideas]
templ removes all redundant whitespaces from the generated output. This makes it hard to see what changed in my testdata fixture outputs. Would it be possible to have an option to disable this behaviour?

**Comment by akfaew** [comment]
Another issue this is causing, is that code inside of:

```
<pre>
a
b
c
</pre>
```

gets rendered as `<pre>a b c</pre>` which makes this three-line code sample become one line.

**Reply by pharrisee** [reply]
Just in case you haven't seen this, it's already been raised as an issue:

https://github.com/a-h/templ/issues/937
